{"url": "https://api.github.com/repos/datalad/datalad/issues/3647", "repository_url": "https://api.github.com/repos/datalad/datalad", "labels_url": "https://api.github.com/repos/datalad/datalad/issues/3647/labels{/name}", "comments_url": "https://api.github.com/repos/datalad/datalad/issues/3647/comments", "events_url": "https://api.github.com/repos/datalad/datalad/issues/3647/events", "html_url": "https://github.com/datalad/datalad/issues/3647", "id": 488501439, "node_id": "MDU6SXNzdWU0ODg1MDE0Mzk=", "number": 3647, "title": "test_annex_ssh flakes out while testing 0.11.x", "user": {"login": "yarikoptic", "id": 39889, "node_id": "MDQ6VXNlcjM5ODg5", "avatar_url": "https://avatars.githubusercontent.com/u/39889?v=4", "gravatar_id": "", "url": "https://api.github.com/users/yarikoptic", "html_url": "https://github.com/yarikoptic", "followers_url": "https://api.github.com/users/yarikoptic/followers", "following_url": "https://api.github.com/users/yarikoptic/following{/other_user}", "gists_url": "https://api.github.com/users/yarikoptic/gists{/gist_id}", "starred_url": "https://api.github.com/users/yarikoptic/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/yarikoptic/subscriptions", "organizations_url": "https://api.github.com/users/yarikoptic/orgs", "repos_url": "https://api.github.com/users/yarikoptic/repos", "events_url": "https://api.github.com/users/yarikoptic/events{/privacy}", "received_events_url": "https://api.github.com/users/yarikoptic/received_events", "type": "User", "site_admin": false}, "labels": [], "state": "closed", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 2, "created_at": "2019-09-03T09:44:48Z", "updated_at": "2020-10-15T13:38:21Z", "closed_at": "2020-10-15T13:38:20Z", "author_association": "MEMBER", "active_lock_reason": null, "body": "https://travis-ci.org/datalad/datalad/jobs/580117784  (but I will restart it now since I think it is just a flaky functioning/test and it had been passing before)\r\n```\r\n======================================================================\r\nFAIL: datalad.support.tests.test_annexrepo.test_annex_ssh\r\n----------------------------------------------------------------------\r\nTraceback (most recent call last):\r\n  File \"/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/nose/case.py\", line 198, in runTest\r\n    self.test(*self.arg)\r\n  File \"/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/datalad/tests/utils.py\", line 921, in newfunc\r\n    return func(*args, **kwargs)\r\n  File \"/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/datalad/tests/utils.py\", line 615, in newfunc\r\n    return t(*(arg + (filename,)), **kw)\r\n  File \"/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/datalad/tests/utils.py\", line 791, in newfunc\r\n    t(*(arg + (uri,)), **kw)\r\n  File \"/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/datalad/tests/utils.py\", line 791, in newfunc\r\n    t(*(arg + (uri,)), **kw)\r\n  File \"/home/travis/virtualenv/python3.6.7/lib/python3.6/site-packages/datalad/support/tests/test_annexrepo.py\", line 1218, in test_annex_ssh\r\n    ok_(exists(socket_1))\r\nAssertionError: None\r\n```\r\n Commit fc66340 ", "closed_by": {"login": "mih", "id": 136479, "node_id": "MDQ6VXNlcjEzNjQ3OQ==", "avatar_url": "https://avatars.githubusercontent.com/u/136479?v=4", "gravatar_id": "", "url": "https://api.github.com/users/mih", "html_url": "https://github.com/mih", "followers_url": "https://api.github.com/users/mih/followers", "following_url": "https://api.github.com/users/mih/following{/other_user}", "gists_url": "https://api.github.com/users/mih/gists{/gist_id}", "starred_url": "https://api.github.com/users/mih/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/mih/subscriptions", "organizations_url": "https://api.github.com/users/mih/orgs", "repos_url": "https://api.github.com/users/mih/repos", "events_url": "https://api.github.com/users/mih/events{/privacy}", "received_events_url": "https://api.github.com/users/mih/received_events", "type": "User", "site_admin": false}, "reactions": {"url": "https://api.github.com/repos/datalad/datalad/issues/3647/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/datalad/datalad/issues/3647/timeline", "performed_via_github_app": null}